fastestimator/op/numpyop/numpyop.py

```diff
@@ -134,6 +134,14 @@ def __init__(self,
         self.fn = fn
         self.in_list = True
 
+    def set_rua_level(self, magnitude_coef: float) -> None:
+        """A method which will be invoked by RUA Op to adjust the augmentation intensity.
+
+        Args:
+            magnitude_coef: Factor to set the range for magnitude of augmentation.
+        """
+        pass
```

Comment:
codacy is correct. Can eliminate pass since there's a doc string here. Also, see above comment on doc string.